so_box_biogeo/diags/estNsq.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from MITgcmutils import rdmds, densjmd95
+from hspython import loadgrid
```

Comment:
Yes, that makes sense. But it is likely that I will use this function quite a lot in the future.
I will put a file with "loadgrid" in this directory for now.